### PR TITLE
Add support for vSCSI pi_instance deployment

### PR DESCRIPTION
**Pull request by yussufsh**
Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by yussufsh** [comment]
cc @hkantare

File: ibm/resource_ibm_pi_instance.go [hunk 1]
```diff
@@ -114,13 +114,19 @@ func resourceIBMPIInstance() *schema.Resource {
...
+			helpers.PIInstanceStorageType: {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Computed:    true,
 				Description: "Storage type for server deployment",
 			},
 
+			helpers.PIInstanceStorageConnection: {
+				Type:        schema.TypeString,
+				Optional:    true,
```
Comment (hkantare) [inline ibm/resource_ibm_pi_instance.go:126]: Can you add validate func if the supported value is only `vSCSI`

Comment (yussufsh) [reply ibm/resource_ibm_pi_instance.go]: done

**Comment by yussufsh** [comment]
Thanks @hkantare

**Pull request merged**